ui/src/serverIO.js

```diff
@@ -154,6 +120,8 @@ class ServerIO {
     });
 
     this.loggingSocket.on('disconnect', (reason) => {
+      console.log('loggingSocket disconnected!'); // eslint-disable-line no-console
```

Comment:
For debugging then I guess ?

Reply:
Yeah, I figured there's one for `connect`, might as well have one for `disconnect`.